DemoApp/android/app/src/main/AndroidManifest.xml

```diff
@@ -6,9 +6,7 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.CAMERA" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
-    <uses-sdk
-        android:minSdkVersion="16"
```

Comment:
I don't think we can change the minimum version for our test app until we've stopped supporting the lower versions

Reply:
Is it a requirement of the new async storage package?

Reply:
It should come from gradle the min and target versions, please do not use Manifest to manage that.

Reply:
It was requested by the build system:
```
ERROR: The minSdk version should not be declared in the android manifest file. You can move the version from the manifest to the defaultConfig in the build.gradle file.
Remove minSdkVersion and sync project
```
I'll remove it from Manifest.